pkg/config/process_test.go

```diff
@@ -70,6 +70,14 @@ func TestProcessDefaultConfig(t *testing.T) {
 			key:          "process_config.windows.use_perf_counters",
 			defaultValue: false,
 		},
+		{
+			key:          "process_config.process_dd_url",
+			defaultValue: DefaultProcessEndpoint,
+		},
+		{
+			key:          "process_config.additional_endpoints",
+			defaultValue: make(map[string][]string),
+		},
```

Comment:
we should probably add a test for the env var override as well

Reply:
Ah, thanks for pointing that out. I went ahead and added one.